README_zh.md

```diff
@@ -0,0 +1,57 @@
+[TOC]
```

Comment:
TOC is not supported by github.
u can use vs code plugin `markdown all in one` to create detailed toc content.